lib/ansible/modules/network/ios/ios_logging.py

```diff
@@ -249,9 +271,20 @@ def parse_level(line, dest):
     return level
 
 
+def parse_trap(line, dest):
+    trap_group = ('emergencies', 'alerts', 'critical', 'errors',
+                  'warnings', 'notifications', 'informational', 'debugging')
+
+    if dest == 'host' and line in trap_group:
+        trap = line
+    else:
+        trap = 'informational'
```

Comment:
why set trap to `informational`?

Reply:
I thought it should be something like default value. So, if value set in playbook exists in trap_group we admit it, and set trap to 'informational' if not. Wrong solution?